File: tugas/soalB/java/TimBola.java
public class TimBola{
    // atribut
    private String nama;
    private String asalNegara;
    private int sejak;
    private String pemain;

    // constructor 
    public TimBola(){}

    // constructor dengan parameter
    public TimBola(String name, String country, int since, String player){
        this.nama = name;
        this.asalNegara = country;
        this.sejak = since;
        this.pemain = player;
    }

    // setter
    public void setNama(String name){
        this.nama = name;
    }
    public void setAsalNegara(String country){
        this.asalNegara = country;
    }
    public void setSejak(int since){
        this.sejak = since;
    }
    public void setPemain(String player){
        this.pemain = player;
    }

    // getter
    public String getNama(){
        return nama;
    }
    public String getAsalNegara(){
        return asalNegara;
    }
    public int getSejak(){
        return sejak;
    }
    public String getPemain(){
        return pemain;
    }
}

File: tugas/soalB/cpp/TimBola.cpp
#include <iostream>
#include <string>

using namespace std;

class TimBola{
    private:
        /* atribut */
        string nama, asalNegara, pemain;
        int sejak;

    public:
        /* constructor */
        TimBola(){
            this->nama = this->asalNegara = this->pemain = "";
            this->sejak = 0;
        }

        /* constructor dengan parameter */
        TimBola(string name, string country, int since, string player){
            this->nama = name;
            this->asalNegara = country;
            this->sejak = since;
            this->pemain = player;
        }

        /* setter */
        void setNama(string name){
            this->nama = name;
        }
        void setAsalNegara(string country){
            this->asalNegara = country;
        }
        void setSejak(int since){
            this->sejak = since;
        }
        void setPemain(string player){
            this->pemain = player;
        }

        /* getter */
        string getNama(){
            return this->nama;
        }
        string getAsalNegara(){
            return this->asalNegara;
        }
        int getSejak(){
            return this->sejak;
        }
        string getPemain(){
            return this->pemain;
        }
        
        ~ TimBola(){}
};

